packages/transport/scripts/protobuf-patches/index.js

```diff
@@ -112,6 +112,7 @@ const TYPE_PATCH = {
     'Features.experimental_features': 'boolean | null',
     'HDNodePathType.node': 'HDNodeType | string',
     'FirmwareUpload.payload': 'Buffer | ArrayBuffer',
+    'EthereumGetAddress.encoded_network': 'ArrayBuffer',
```

Comment:
we may only need this part from all the protobuf related changes and also resulting changes from types

Reply:
Yes, I updated, that is the way it is now 4c14e8fcae0f37c774f972e493e103350952880f